generators/app/templates/site-uris-test.js

```diff
@@ -38,11 +27,7 @@ describe('price-finder for <%= upperCaseSite %> Store URIs', () => {
     it('should respond with a price, and the right category and name for findItemDetails()', (done) => {
       priceFinder.findItemDetails(uri, (err, itemDetails) => {
         expect(err).toBeNull();
-        expect(itemDetails).toBeDefined();
-
-        verifyPrice(itemDetails.price);
-        verifyName(itemDetails.name, '');
-        verifyCategory(itemDetails.category, '');
+        verifyItemDetails(itemDetails, 'PixelJunk™ Monsters', 'Video Games');
```

Comment:
Please remove 'PixelJunk™ Monsters' and 'Video Games' and instead leave them as empty strings ('').